Makefile

```diff
@@ -0,0 +1,31 @@
+.PHONY: all build-policy build-plugin test e2e clean fmt
+
+# Get the plugin path from the SDK
+KUBEWARDEN_PLUGIN := $(shell node --input-type=module -e "import sdk from 'kubewarden-policy-sdk'; console.log(sdk.pluginPath)")
```

Comment:
I think we do not need this. We can assume that the file will always be under the `node_modules` directory and point to it directly.